lib/imap/sync.rb

```diff
@@ -407,7 +407,7 @@ def update_email(incoming_email)
     end
 
     def tagging_enabled?
-      SiteSetting.tagging_enabled && SiteSetting.allow_staff_to_tag_pms
+      SiteSetting.tagging_enabled && Guardian.new(Discourse.system_user).can_tag_pms?
```

Comment:
```suggestion
      SiteSetting.tagging_enabled && Guardian.new(Discourse.system_user).can_tag_pms?
```

Reply:
No difference here, anything in specific you are pointing out?